ilastik/workflows/newAutocontext/newAutocontextWorkflow.py

```diff
@@ -266,25 +278,47 @@ def checkConstraints(*_):
 
         for ( upstreamPcApplet,
               downstreamFeaturesApplet,
-              downstreamPcApplet ) in zip( upstreamPcApplets, 
-                                           downstreamFeatureApplets, 
+              downstreamPcApplet ) in zip( upstreamPcApplets,
+                                           downstreamFeatureApplets,
                                            downstreamPcApplets ):
-            
+
             opUpstreamClassify = upstreamPcApplet.topLevelOperator.getLane(laneIndex)
             opDownstreamFeatures = downstreamFeaturesApplet.topLevelOperator.getLane(laneIndex)
             opDownstreamClassify = downstreamPcApplet.topLevelOperator.getLane(laneIndex)
 
             # Connect label inputs (all are connected together).
             #opDownstreamClassify.LabelInputs.connect( opUpstreamClassify.LabelInputs )
-            
+
             # Connect data path
-            #assert opData.Image.meta.dtype == numpy.uint8, "Raw Data must be uint8, not {}".format( opData.Image.meta.dtype )
+<<<<<<< HEAD
```

Comment:
please resolve merge conflict

Reply:
Arg! Yes, I will undo this - I have accidentally merged that in (together with the autopep) while figuring out that I had an `autocontext_uint16` and `feature_autocontext_uint16`  branch.

Reply:
Arg! I noticed that I created the `feature_autocontext_uint16` because the `autocontext_uint16` had the autopep in there... I need to investigate how to best merge the new  `autocontext_uint16` features into  `feature_autocontext_uint16`  without the autopep stuff...

Reply:
I think I did it! :tada: